.gitignore

```diff
@@ -51,8 +51,40 @@ wheels/
 # Checksums
 *.sha256
 
-# Data files (add specific patterns as needed)
-# *.csv
-# *.xlsx
-# *.parquet
-# data/
+# Data files - common formats
+*.csv
+*.xlsx
+*.xls
```

Comment:
Perhaps could include `.xlsm`, `.docx`, `.doc` and `.pdf` as well, to be extra "safe"